validator/validator-main.protoascii

```diff
@@ -1743,8 +1743,14 @@ attr_lists: {
 }
 attr_lists: {
   name: "svg-xlink-attributes"
+  attrs: {
```

Comment:
Did you try setting alternative_names: "href" on the xlink:href attribute?
Slightly shorter and would make it clearer that it means the same. I don't remember whether it would avoid allowing both on the same tag.